core/src/test/java/org/mule/tck/junit4/AbstractMuleTestCase.java

```diff
@@ -55,6 +55,11 @@
     public static final String TEST_TIMEOUT_SYSTEM_PROPERTY = "mule.test.timeoutSecs";
 
     /**
+     * Default timeout for waiting for responses
+     */
+    public static final int RECEIVE_TIMEOUT = 5000;
```

Comment:
I only moved this up from AbstractMuleContextTestCase, AbstractMuleContextTestCase is also a base unit test case...